src/navigation/Link/Link.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { PropsWithChildren } from "react";
+import styles from "./Link.module.css";
+import { Plus } from "../../icons";
+import classNames from "classnames/bind";
+const cx = classNames.bind(styles);
+
+interface LinkProps {
+  to: string;
+  id?: string;
+  target?: "_blank" | "_self" | "_parent" | "_top";
+  disabled?: boolean;
+  onClick?: (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => void;
+  children: React.ReactNode;
```

Comment:
**nitpick:** No need for declaring `children` here as we use `PropsWithChildren` already